### PR TITLE
ref(ts) Convert Indicators to typescript.

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/alerts/toastIndicator.tsx [hunk 1]
```diff
@@ -42,7 +43,7 @@ Toast.defaultProps = {
   }),
 };
 
-const Icon = styled('div')`
+const Icon = styled('div', {shouldForwardProp: p => p !== 'type'})<{type: string}>`
   margin-right: 6px;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/alerts/toastIndicator.tsx:47]:
```suggestion
  margin-right: ${space(0.75)};
```

File: src/sentry/static/sentry/app/components/alerts/toastIndicator.tsx [hunk 2]
```diff
@@ -60,7 +61,7 @@ const Undo = styled('div')`
   color: ${p => p.theme.gray500};
   padding-left: 16px;
   margin-left: 16px;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/alerts/toastIndicator.tsx:63]:
```suggestion
  margin-left: ${space(2)};
```

File: src/sentry/static/sentry/app/components/alerts/toastIndicator.tsx [hunk 3]
```diff
@@ -60,7 +61,7 @@ const Undo = styled('div')`
   color: ${p => p.theme.gray500};
   padding-left: 16px;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/alerts/toastIndicator.tsx:62]:
```suggestion
  padding-left: ${space(2)};
```

File: src/sentry/static/sentry/app/components/alerts/toastIndicator.tsx [hunk 4]
```diff
@@ -75,8 +76,14 @@ const StyledLoadingIndicator = styled(LoadingIndicator)`
   }
 `;
 
-function ToastIndicator({indicator, onDismiss, className, ...props}) {
-  let icon;
+type Props = {
+  className?: string;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/alerts/toastIndicator.tsx:80]: Please could you move this prop to the end of the object? 🙏

**Review by evanpurkhiser — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]
🎉

**Pull request merged**